### PR TITLE
Django 3 support

### LINKED ISSUE #263: Django 3.0 Support
Hi *,

thanks for providing this really useful fulltext search app :).

Since django 3.0 was released yesterday I tried to run it with django-watson but got this traceback:

```
  File "/Users/jochen/.pyenv/versions/3.8.0/envs/homepage/lib/python3.8/site-packages/watson/models.py", line 9, in <module>
    from django.utils.encoding import python_2_unicode_compatible, force_text
ImportError: cannot import name 'python_2_unicode_compatible' from 'django.utils.encoding' (/Users/jochen/.pyenv/versions/3.8.0/envs/homepage/lib/python3.8/site-packages/django/utils/encoding.py)
/Users/jochen/.pyenv/versions/3.8.0/envs/homepage/lib/python3.8/site-packages/django/core/servers/basehttp.py changed, reloading.
```

Ah ok, those [compatibility apis](https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis) have been removed in django 3.0. Created a [naive pull request](https://github.com/etianen/django-watson/pull/264) to fix it, let's see how much travis will complain :).

best regards,
jochen

### PR DESCRIPTION
**Pull request by biozz**
I am following @ephes with his pull request #264 with almost the same stuff: adding support for django 3.

This one is intended to resolve #263.

I also fixed one test file and changed travis ci test matrix according to the one you propsed in #264 - https://github.com/etianen/django-reversion/blob/master/.travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,32 +4,25 @@ python:
...
   - 3.7
   - 3.6
-  - 3.5
-  - 2.7
 cache: pip
 env:
   global:
     - PYTHONWARNINGS=default,ignore::PendingDeprecationWarning,ignore::ResourceWarning
+    - DJANGO_DATABASE_USER_POSTGRES=postgres
+    - DJANGO_DATABASE_USER_MYSQL=travis
```
Comment (etianen) [inline .travis.yml]: I don't think these environmental variables are used by django-watson. I take it they're taken from the `travis.yml` in `django-reversion`?

So I think these can be taken out.

Comment (biozz) [reply .travis.yml]: You are right, I copied them from `django-reversion` for the sake of full travis config match, but they are not affecting the build. They are removed now.

**Comment by biozz** [comment]
I just noticed that `README.markdown` contains some mentions of Django and Python versions, which are going to be outdated after the merge. Do you want them changed in this PR as well?

**Comment by etianen** [comment]
Please!

**Comment by etianen** [comment]
This is perfect. I'll push out a release today. Thanks!

**Comment by etianen** [comment]
Released as 1.5.4

**Pull request merged**